app/controllers/api/v2/smart_proxy_hosts_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Api
+  module V2
+    class SmartProxyHostsController < V2::BaseController
```

Comment:
I don't see a GET implemented. Is that implicit?

Reply:
https://github.com/theforeman/foreman/pull/8632#discussion_r665183203